packages/ui-responsive/src/Responsive/__tests__/Responsive.test.tsx

```diff
@@ -190,4 +190,35 @@ describe('<Responsive />', async () => {
 
     expect(responsiveDiv.style.display).to.equal('inline-flex')
   })
+
+  it('should provide correct props in case of multiple breakpoints in query', async () => {
+    const renderSpy = spy()
+    const props = {
+      small: { withBorder: true, background: 'transparent' },
+      medium: { options: [1, 2, 3], icons: { edit: true, flag: false } },
+      large: { margin: 'small', label: 'hello world', describedBy: 'fakeId' }
+    }
+    await mount(
+      <div style={{ width: 500, height: 400 }}>
+        <Responsive
+          props={props}
+          query={{
+            small: { maxWidth: 300, maxHeight: 200 },
+            medium: { maxWidth: 800, maxHeight: 500 },
```

Comment:
wouldn't a height of 199 match both small and medium here?

Reply:
Yes, both would be matched for a heigth of 199, furthermore a warning would be emitted if there were properties with the same keys. In this testcase (width: 500, height: 400), the medium labelled props should be picked up.

Reply:
Wouldn't both small and medium props be picked up in some cases at a height of 199?

Reply:
yes, they would. But this is not an issue, one can write queries with overlapping intervals. A warning (`The prop ${prop} is defined at 2 or more breakpoints...`) will be emitted if one defines a the same prop for overlapping intervals. So one can write something like:
```json
props=
  {small: {border:"black"}}
  {medium: {margin:"small"}}
query=
  {small: {minHeight: 200}
  {medium: {minHeight: 300}
```
And if `height` is above 299 pixels both props (`border` and `margin`) are passed to the render method without an issue. 
But if one defines overlapping query intervals where the same props are defined will result in a warning because the prop value is ambiguous, for example:
```json
props:
  {small: {border:"black"}}
  {medium: {border:"red"}}
query:
  {small: {minHeight: 200}
  {medium: {minHeight: 300}
```
This will result in a warning because InstUI cannot decide what value the `border` should have above 299 pixels.